### PR TITLE
Contact form feature

### PR DESCRIPTION
**Pull request by OyePriscilla**
## Html Validations
The input boxes validated with the basic html5 validations technique

## Data Collection
FormSpree Service was used to collect data

## Display
Contact form display with FlexBox.

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages.
:heavy_check_mark: Clean and organized code. 

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below under the review._

 #### I will leave the comment here, since your `Readme file` is not accessible for this PR ####
- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your poject. Since this is a group team project, please ensure you update the `Authors section` with information pertaining to both team members.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ This Project is created purposely to learn how to build Portfolio.
 
 ## Requirements
 * Create a new GitHub Repo and a new branch with GitHub flow
-*  Setup Linter for HTML and CSS
+* Setup Linter for HTML and CSS
+* Add Form validation 
 
 ## Getting Started
```
Comment (danmainah) [inline README.md]: - [ ] Kindly recheck your readme file, I am afraid the information in your getting started section is not the required one, the getting started section is supposed to help visitors to your repo know how to cone and run your project locally, what they will need and the steps to follow.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @OyePriscilla, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
Thanks @danmainah for the review. Please the review is confusing, you
stated that the information in my ReadMe file getting started is not the
required one but you commented on a different line. Can you please clarify
it a little further? Am I to work on the *getting started* or  this "*
Setup Linter for HTML and CSS, * Add Form validation ".

Thanks
@OyePriscilla

**Comment by danmainah** [comment]
@OyePriscilla  sorry for the confusion , you are to work on the getting started section

**Review by muneebulrehman — APPROVED** [review]
Hi @OyePriscilla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️

## Highlights
- No linter errors
- Good HTML structure
- Good follow-up with previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- It would be great if you would update the Author's section of your README file. There are two authors mentioned, but you are alone working on the project. So, kindly update it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**